lib/mutate/requireMutations.js

```diff
@@ -1,12 +1,15 @@
 import { remote } from "electron";
-
+const muts = remote.require("./mutations");
 let requireMutations = () => [];
 
-// Don't run in test env
 if (process.env.NODE_ENV !== "test") {
```

Comment:
use guard pattern here? ;)

Reply:
Would if I could 😭

`exports` have to be defined at the bottom of the file, so I can't do this:

``` js 
if (process.env.NODE_ENV === "test") {
  export default [];
}

// ... do some stuff
export default requireMutations;
```